packages/react-components/react-charts-preview/stories/VerticalBarChart/VerticalBarChartDefault.stories.tsx

```diff
@@ -1,9 +1,10 @@
 import * as React from 'react';
-import { VerticalBarChart, IVerticalBarChartProps, IVerticalBarChartDataPoint } from '../../src/VerticalBarChart';
+import { VerticalBarChart, IVerticalBarChartProps, IVerticalBarChartDataPoint, ILineChartLineOptions} from '../../src/VerticalBarChart';
 import { DefaultPalette } from '@fluentui/react/lib/Styling';
 import { ChoiceGroup, IChoiceGroupOption } from '@fluentui/react/lib/ChoiceGroup';
 import { Checkbox } from '@fluentui/react/lib/Checkbox';
 import { Switch } from '@fluentui/react-components';
+import { IRenderFunction } from '@fluentui/react/lib/Utilities';
```

Comment:
don't use v8 functions

Reply:
Okay. will remove this

Reply:
> don't use v8 functions

So is there any alternate function for this?

Reply:
To be fixed separately in another PR.